ner_v2/detectors/numeral/number/standard_number_detector.py

```diff
@@ -46,7 +52,7 @@ def detect_number(self, text):
         return number_list, original_list
 
     @staticmethod
-    def _get_numerals_list(numerals_text):
+    def _get_list_from_pipe_sep_string(numerals_text):
```

Comment:
Can be moved to `ner_v2/detectors/numeral/utils.py`

Reply:
Done